common/gtk-3.0/3.20/sass/_common.scss

```diff
@@ -1616,18 +1617,18 @@ menu,
   separator,
   .csd & separator {
     margin: 2px 0;
-    background-color: $_menu_bg;
+    background-color: $_sp_menu_bg;
   }
 
   // Firefox workaround
   .separator:not(label),
-  .csd & .separator:not(label) { color: $_menu_bg; }
+  .csd & .separator:not(label) { color: $_sp_menu_bg; }
   // Firefox workaround end
 
   menuitem {
-    min-height: 16px;
+    min-height: 13px;
```

Comment:
The color change above makes sense to me, but could you explain in short why to also change the height and padding?

Reply:
Because it wasn't in the middle it has more padding on top for some reason I don't get it.